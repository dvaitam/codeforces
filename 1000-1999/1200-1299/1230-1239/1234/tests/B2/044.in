17 8
19 5 18 6 9 1 14 19 2 12 14 13 10 1 3 3 1
