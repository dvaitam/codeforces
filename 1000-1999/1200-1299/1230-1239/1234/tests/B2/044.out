8
3 10 13 12 2 14 1 9
